content/source/docs/enterprise/private/faq.html.md

```diff
@@ -310,7 +192,7 @@ Now, if you keep the `backend.tf` file in scope when you run `terraform` operati
 
 ## Private Terraform Enterprise Architecture
 
-This document describes aspects of the architecture of Private Terraform Enterprise as deployed via the AMI.
+This document describes aspects of the architecture of Private Terraform Enterprise.
```

Comment:
As I understand it, this diagram is a little outdated by still applies to installer version https://www.terraform.io/docs/enterprise/private/faq.html#private-terraform-enterprise-architecture